flow/controllers/base_controller.py

```diff
@@ -90,6 +98,12 @@ def get_action(self, env):
         float
             the modified form of the acceleration
         """
+        # clear the current stored accel_no_noise_no_failsafe of this vehicle None
+        env.k.vehicle.update_accel(self.veh_id, None, noise=False, failsafe=False)
+        env.k.vehicle.update_accel(self.veh_id, None, noise=False, failsafe=True)
+        env.k.vehicle.update_accel(self.veh_id, None, noise=True, failsafe=False)
+        env.k.vehicle.update_accel(self.veh_id, None, noise=True, failsafe=True)
```

Comment:
That doesn't seem necessary since all 4 accel values are set afterwards

Reply:
they need to be set in case the logic is short-circuited (i.e. a few lines below, `if accel is None: return None`)

Reply:
right, didn't see that